src/main/java/bio/terra/janitor/service/pubsub/TrackedResourceSubscriber.java

```diff
@@ -25,13 +25,16 @@
 
   private final TrackResourcePubsubConfiguration trackResourcePubsubConfiguration;
   private final JanitorService janitorService;
+  private final ObjectMapper objectMapper;
 
   @Autowired
   TrackedResourceSubscriber(
       TrackResourcePubsubConfiguration trackResourcePubsubConfiguration,
-      JanitorService janitorService) {
+      JanitorService janitorService,
+      @Qualifier("objectMapper") ObjectMapper objectMapper) {
```

Comment:
what's up with the `@Qualifier`?

Reply:
It tells spring to use this ObjectMapper we created. https://github.com/DataBiosphere/terra-resource-janitor/blob/master/src/main/java/bio/terra/janitor/app/configuration/ApplicationConfiguration.java#L27